tmp/build/patch-charts.sh

```diff
@@ -123,6 +123,10 @@ function patchTemplates() {
     xargs sed -i -e 's/^\(.*\)release:\(.*\)$/\1maistra-version: '${MAISTRA_VERSION}'\
 \1release:\2/'
 
+# MAISTRA-506 add a namespace label for deployment specs
+find ${HELM_DIR} -name "*.yaml" -o -name "*.yaml.tpl" | xargs grep -Hl 'kind: Deployment' |\
+  xargs sed -i -e '/^spec:/,$ { /template:$/,$ { /metadata:$/,$ { /labels:$/,$ s/^\(.*\)release:\(.*\)$/\1namespace: {{ .Release.Namespace }}\n\1release:\2/ } } }'
+
```

Comment:
@knrc what about `control-plane` for the label name?

Reply:
@jwendell +1 although it might be better to prefix it with maistra so we don't accidentally clash in future, i.e. maistra-control-plane

Reply:
@knrc done